draft-ar-pquip-pqc-engineers.md

```diff
@@ -231,9 +231,9 @@ These algorithms are not a drop-in replacement for classical asymmetric cryptogr
 The fourth-round of the NIST process focuses only on KEMs. The goal of that round is to select an althernative algorithm that is based on different hard problem than Kyber.
 The candidates still advancing for standardization are:
 
-* [Classic McEliece](https://classic.mceliece.org/): Based on the hardness of syndrome decoding of Goppa codes
-* [BIKE](https://bikesuite.org/): Based on the the hardness of syndrome decoding of QC-MDPC codes.
-* [HQC](http://pqc-hqc.org/) : Based on the hardness of syndrome decoding of Hamming Quasi-cyclic codes.
+* [Classic McEliece](https://classic.mceliece.org/): Based on the hardness of syndrome decoding of Goppa codes. Goppa codes are a class of error-correcting codes that can correct a certain number of errors in a transmitted message. The decoding problem involves recovering the original message from the received noisy codeword.
+* [BIKE](https://bikesuite.org/): Based on the the hardness of syndrome decoding of QC-MDPC codes. Quasi-Cyclic Moderate Density Parity Check (QC-MDPC) code are a class of error correcting codes that leverages bit flipping technique to efficiently correct the Hamming weight proportional to square root of the code length.
+* [HQC](http://pqc-hqc.org/) : Based on the hardness of syndrome decoding of Quasi-cyclic BCH codes. BCH are a class of cyclic error-correcting codes that are constructed using polynomials in the Galois field and provide a great generalization of the Hamming code for multiple bit-error correction. 
```

Comment:
I propose the following minor update: 
BCH codes are a class of cyclic error-correcting codes constructed using polynomials in the Galois field. They offer a significant generalization of the Hamming code, enabling the correction of multiple bit errors.

Reply:
Addressed.